tests/ff-server-sdk-test/ff-server-sdk-test.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net7.0</TargetFrameworks>
+    <TargetFrameworks>net7.0;net8.0</TargetFrameworks>
     <RootNamespace>ff_server_sdk_test</RootNamespace>
     <LangVersion>9.0</LangVersion>
     <IsPackable>false</IsPackable>

```

Comment:
can be dripped, test sdk already does this

Reply:
I removed net7.0 from here.

Reply:
Removed the net7.0 reference

Reply:
was referring to the `IsPackable`, but agree that's a good change too ;)